tests/e2e/tests/checkout/checkout.page.ts

```diff
@@ -245,4 +254,96 @@ export class CheckoutPage {
 			this.page.locator( '.wc-block-order-confirmation-billing-wrapper' )
 		).toBeVisible();
 	}
+
+	async verifyOrderConfirmationDetails(
+		currentPage: Page,
+		toBeVisible = true
+	) {
+		const statusSection = currentPage.locator(
+			'[data-block-name="woocommerce/order-confirmation-status"]'
+		);
+		const summarySection = currentPage.locator(
+			'[data-block-name="woocommerce/order-confirmation-summary"]'
+		);
+		const totalsSection = currentPage.locator(
+			'[data-block-name="woocommerce/order-confirmation-totals"]'
+		);
+		const shippingAddressSection = currentPage.locator(
+			'[data-block-name="woocommerce/order-confirmation-shipping-address"]'
+		);
+		const billingAddressSection = currentPage.locator(
+			'[data-block-name="woocommerce/order-confirmation-billing-address"]'
+		);
+
+		if ( toBeVisible ) {
```

Comment:
Does this take care of the partial view for guests? If it doesn't and its not needed, just add a note to the method that it only handles the full logged in view.